language/move-vm/runtime/src/interpreter.rs

```diff
@@ -251,18 +271,28 @@ impl Interpreter {
     ///
     /// Native functions do not push a frame at the moment and as such errors from a native
     /// function are incorrectly attributed to the caller.
-    fn make_call_frame(&mut self, func: Arc<Function>, ty_args: Vec<Type>) -> VMResult<Frame> {
+    fn make_call_frame(
+        &mut self,
+        func: Arc<Function>,
+        ty_args: Vec<Type>,
+        paranoid: bool,
```

Comment:
I think you can directly access `self.runtime_config.paranoid_type_checks` without passing the flag in, no?